modules/akmods/akmods.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -oue pipefail
+
+BASED_IMAGE=$(echo "${BASE_IMAGE}")
```

Comment:
What's the point of this? The `BASE_IMAGE`and `BASED_IMAGE` variable are identical, no?

Reply:
You're right, there is no need for this.

I can correct this in another PR.